workflow-ui/modal-android/src/main/java/com/squareup/workflow1/ui/modal/ModalViewContainer.kt

```diff
@@ -62,7 +62,12 @@ public open class ModalViewContainer @JvmOverloads constructor(
     initialViewEnvironment: ViewEnvironment
   ): DialogRef<Any> {
     val view = initialViewEnvironment[ViewRegistry]
-        .buildView(initialModalRendering, initialViewEnvironment, this)
+        .buildView(
```

Comment:
This change and a couple of others like it are because I deleted a convenience overload of `buildView` that caused nothing but trouble, and which in practice no one but the library will ever use.

Reply:
Yes! Figured that's what was going on, and I've wanted to do that in the past too. Thanks!